### PR TITLE
B shapemerge config

### PR DESCRIPTION
**Pull request by ndjc**
Changes to configuration management to merge shapes into one file, and correct various warnings from linters.

### CODE REVIEW

File: specs/config/config-resources.html [hunk 1]
```diff
@@ -228,9 +228,9 @@ <h2>Supported Operations on Components</h2>
     <li>POST to the configuration LDPC for a component to create a new stream for that component, optionally specifying an initial baseline;
       if an initial baseline is not specified, a real or notional empty baseline is implied as the starting point for the stream</li>
   </ul>
-  <p>Servers SHOULD support the Prefer header for GET on LDPCs specifying PreferMinimalContainer:
+  <p>Servers SHOULD support the Prefer header for GET on LDPCs specifying PreferMinimalContainer:</p>
```
Comment (berezovskyi) [inline specs/config/config-resources.html:231]: Could you please enable prettier linter check in CircleCI config for CfgM spec, please?

Comment (ndjc) [reply specs/config/config-resources.html:231]: Will do, in the near future when I work on the spec itself. At the moment, I'm just merging the shapes to address that issue. The spec needs several things to being it up to date, including conformance clause work, using the right URIs and most recent ReSpec, etc.

Comment (ndjc) [reply specs/config/config-resources.html:231]: In other words, the current draft would seriously fail most checks!

Comment (berezovskyi) [reply specs/config/config-resources.html:231]: To ease the pain, I'd suggest to enable format on save via https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode. This requires you joining the dark side ofc and replacing Notepad++ with VS Code.

Let's merge now as is.

Finally, Prettier is only checking well-formedness and enforces formatting of the HTML code, not much more than that.

**Comment by ndjc** [comment]
Fixed license and URI for shapes - will address prettier and other concerns with actual spec in a separate branch & PR.

**Pull request merged**